build-gluster-org/jobs/smoke-template.yml

```diff
@@ -0,0 +1,76 @@
+- project:
+    name: 'smoke'
+    os_type:
+      - 32-bit-build:
+          node_name: fedora32
+          description: smoke job for 32 bit build - https://github.com/gluster/build-jobs/blob/main/build-gluster-org/scripts/32-bit-build-smoke.sh
+          keyword: 32-bit-build 
+          timeout: 30
+          script_name: 32-bit-build-smoke
+          disabled_var: true
+      - fedora:
+          node_name: fedora32
+          description: Compile the code with GCC 10 - https://github.com/gluster/build-jobs/blob/main/build-gluster-org/scripts/fedora-smoke.sh
+          keyword: fedora smoke 
+          timeout: 30
+          script_name: fedora-smoke 
+          blacklist_branch: release-.*
+      - freebsd:
+          node_name: freebsd_build 
+          description: GlusterFS pre-commit smoke test on FreeBSD - https://github.com/gluster/build-jobs/blob/main/build-gluster-org/scripts/freebsd-smoke.sh
+          keyword: freebsd smoke 
+          timeout: 45
+          script_name: freebsd-smoke 
+
+    jobs:
+      - 'gh_{os_type}-{name}'
+
+- job-template:
+    name: 'gh_{os_type}-{name}'
+    node: '{node_name}'
+    description: '{description}'
```

Comment:
Could we automatically add the link to the script here ? Since they are all the same, except script name, this would reduce cut and paste.

Reply:
Updated the description.